deploy/ui/form.json

```diff
@@ -246,6 +246,12 @@
                   "jsonPath": "$.spec.auth.ldap.bindCredential",
                   "description": "LDAP Credentials used for authentication of bindDN"
                 },
+                {
+                  "label": "LoginModule Flag",
+                  "type": "text",
+                  "jsonPath": "$.spec.auth.ldap.loginModule",
+                  "description": "A flag to set login module to optional. The default value is required"
```

Comment:
should the type be set to `dropDown` as seen here?
https://github.com/kiegroup/kie-cloud-operator/blob/8d35c107d0d110be689de7978f8af237a9b7cfeb/deploy/ui/form.json#-L18-L20